screens/Challenges/IndividualChallenge.js

```diff
@@ -13,12 +13,17 @@ import Colors from "../../constants/Colors";
 import KitBackgroundScreen from "../../components/KitBackgroundScreen";
 export default function IndividualChallenge(props) {
   const [rating, setRating] = useState("");
+  const [challenge, setChallenge] = useState({});
+
+  useEffect(()=>{
+    setChallenge(props.navigation.getParam('challenge'))
```

Comment:
Did you ever figure out the bug where it wasn't getting the correct challenge id?